pyrometheus/chem_expr.py

```diff
@@ -204,20 +204,20 @@ def wilke_mixture_rule_expr(sol: ct.Solution, sp, x, mu):
         8*(1 + (w[sp]/w[j]))
     ) for j in range(sol.n_species)])
 
-    
-def species_mixture_rule_expr(sol: ct.Solution, sp, mmw, x, Dij):
+
+def species_mixture_rule_expr(sol: ct.Solution, sp, mmw, x, d_ij):
```

Comment:
Rename to `species_mass_diff_mixture_rule_expr`. Also, I prefer `diff_ij`, than simply `d_ij`.

Reply:
Done.